src/valhalla_pack_elevation.cc

```diff
@@ -0,0 +1,165 @@
+#include <vector>
```

Comment:
so long story here... in `valhalla_build_elevation` i wanted to have a switch to turn the files into lz4 while you were downloading. simple enough but i needed a program that wrote this format. it seems totally annoying to have to write one in c++ because then you have to have the program installed, or at the very least built and in your `PATH`. unfortunately, that is what this program is.

see, the `liblz4-dev` package has version r131 for the c++ bindings. the `python-lz4` package looks to be old crusty r114. these two are not compatible, nor does one have the ability to choose the compression level in the python binding. ok i said, fine... i'll just pip install the latest one, surely its compatible. nope, its version 1.8.0. it seems the lz4 project has had some compatibility changes throughout its development.

so faced with defeat i made a small program to transcode from raw or gzip to lz4 and also validate already made lz4 files. :(